docs/Reference/web3js-eea-Methods.md

```diff
@@ -0,0 +1,144 @@
+description: web3js-eea methods reference
+<!--- END of page meta data -->
+
+Use the [web3.js-eea library](https://github.com/PegaSysEng/eeajs) to [create and send 
+private transactions](../Privacy/Private-Transactions/eeajs.md).
+
+## Options Parameter 
+
+The Options parameter is used by: 
+
+* [`generatePrivacyGroup`](#generateprivacygroup)
+* [`getTransactionCount`](#gettransactioncount)
+* [`sendRawParameter`](#sendrawtransaction)
+
+The Options parameter has the following properties: 
+
+* `privateKey`: Ethereum private key with which to sign the transaction
+* `privateFrom` : Orion public key
+* `privateFor` : Orion public keys of recipients
+* `nonce` : Optional. If not provided, calculated using [`eea_getTransctionCount`](../Reference/Pantheon-API-Methods.md).
+* `to` : Optional. Contract address to which to send the transaction. Do not specify for contract deployment transactions
```

Comment:
Contract address to send the transaction to.

Reply:
Fixed